reana_db/models.py

```diff
@@ -145,14 +146,19 @@ def _get_health_status(usage, limit):
 
         quota_usage = 0
         quota_limit = 0
+        quota_human_readable_usage = ""
         for user_resource in self.resources:
             if user_resource.resource.type_ == resource_type:
                 quota_usage += user_resource.quota_used
                 quota_limit += user_resource.quota_limit
+                quota_human_readable_usage = ResourceUnit.human_readable_unit(
+                    user_resource.resource.unit, user_resource.quota_used
+                )
```

Comment:
> Now I wonder what would happen if a user had more than one resource of a specific resource_type, and each of those with a different unit. This accumulation would be wrong, wouldn't it? Because we'd be adding quota_useds and quota_limits no matter the unit. For now it's fine because we only have one unit per resource_type, but this might bring problems if we add other resources/units. Just dumping my thoughts to beware in the future, it doesn't need to be addressed now, as long as we don't increase resource units.

This is true, it is a shortcoming of a very flexible design... maybe we just create an issue to take into account once we add a new resource?